taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -105,12 +104,10 @@ const ListEntryChildren = ({url, setUpdateNodeChildren}) => {
                         onKeyPress={(e) => { (e.keyCode === ENTER_KEYCODE) && handleAddChild(e) }} 
                         onChange={(e) => { 
                             setNewLanguageCode(e.target.value);
-                            const validateBool = ISO6391.validate(e.target.value);
-                            validateBool ? setisValidLC(true) : setisValidLC(false);
-                            validateBool ? setBtnDisabled(false) : setBtnDisabled(true);
+                            setIsValidLanguageCode(ISO6391.validate(e.target.value));
```

Comment:
could this state be replaced by a derived state? `const isValidLanguageCode = ISO...validate(newLanguageCode)`

Reply:
I think `isValidLanguageCode` should be a state, since the Button component needs to be disabled/enabled based on it.

Reply:
yes, think it would be ok to have it as derived and thus we can get rid of one state

Reply:
Yep, you're right. I made the changes.